idea/idea-analysis/src/org/jetbrains/kotlin/idea/references/referenceUtil.kt

```diff
@@ -281,7 +279,16 @@ fun KtExpression.readWriteAccessWithFullExpression(useResolveForReadWrite: Boole
 }
 
 fun KtReference.canBeResolvedViaImport(target: DeclarationDescriptor, bindingContext: BindingContext): Boolean {
-    if (this is KDocReference) return element.getQualifiedName().size == 1
+    if (this is KDocReference) {
+        val qualifier = element.getQualifier() ?: return true
+        return if (target.isExtension) {
+            val elementHasFunctionDescriptor = element.resolveMainReferenceToDescriptors().find { it is FunctionDescriptor } != null
```

Comment:
It would be better to use `any { it is FunctionDescriptor }` rather `find { it is FunctionDescriptor } != null`